backend/de.metas.printing/de.metas.printing.base/src/main/java/de/metas/printing/model/validator/AsyncBatchNotificationHelper.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * #%L
+ * de.metas.printing.base
+ * %%
+ * Copyright (C) 2022 metas GmbH
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program. If not, see
+ * <http://www.gnu.org/licenses/gpl-2.0.html>.
+ * #L%
+ */
+
+package de.metas.printing.model.validator;
+
+import ch.qos.logback.classic.Level;
+import de.metas.async.AsyncBatchId;
+import de.metas.async.api.IAsyncBatchBL;
+import de.metas.async.api.IAsyncBatchListeners;
+import de.metas.async.api.IWorkPackageQueue;
+import de.metas.async.model.I_C_Async_Batch;
+import de.metas.async.processor.IWorkPackageQueueFactory;
+import de.metas.logging.LogManager;
+import de.metas.organization.ClientAndOrgId;
+import de.metas.printing.async.spi.impl.PrintingQueuePDFConcatenateWorkpackageProcessor;
+import de.metas.printing.model.I_C_Printing_Queue;
+import de.metas.util.Loggables;
+import de.metas.util.Services;
+import lombok.NonNull;
+import lombok.experimental.UtilityClass;
+import org.adempiere.ad.dao.IQueryBL;
+import org.adempiere.ad.dao.IQueryBuilder;
+import org.adempiere.ad.dao.IQueryFilter;
+import org.adempiere.ad.dao.impl.TypedSqlQueryFilter;
+import org.adempiere.model.InterfaceWrapperHelper;
+import org.adempiere.service.ISysConfigBL;
+import org.slf4j.Logger;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+
+import static de.metas.async.Async_Constants.C_Async_Batch_InternalName_AutomaticallyInvoicePdfPrinting;
+
+@UtilityClass
+public class AsyncBatchNotificationHelper
```

Comment:
auch..... this cannot be a utility class with if contains all those services hooked there.

Consider a regular class which u instantiate when u needed it.....
or consider moving those methods in other BLs.